src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Controllers/AuthenticationController.cs

```diff
@@ -369,6 +424,26 @@ private async Task<ActionResult> AuthenticateMaskinportenToken(string originalTo
             }
         }
 
+        private EnterpriseUserCredentials DecodeEnterpriseUserHeader(string encodedCredentials, string orgNumber)
+        {
+            try
+            {
+                byte[] decodedCredentials = Convert.FromBase64String(encodedCredentials);
+                string decodedString = Encoding.UTF8.GetString(decodedCredentials);
+
+                string[] decodedStringArray = decodedString.Split(":");
+                string usernameFromRequest = decodedStringArray[0];
+                string password = decodedStringArray[1];
+
+                EnterpriseUserCredentials credentials = new EnterpriseUserCredentials { UserName = usernameFromRequest, Password = password, OrganizationNumber = orgNumber };
+                return credentials;
+            }
+            catch (Exception)
```

Comment:
This try/catch doesn't do anything besides rethrowing, so just remove.

Reply:
Good point, this is redundant. Removed in latest commit.

Reply:
An alternative return value instead of using a tuple:
`Task<ActionResult<UserAuthenticationResult>>`